k2/csrc/fsa_algo.h

```diff
@@ -158,6 +163,21 @@ Fsa LinearFsa(const Array1<int32_t> &symbols);
  */
 FsaVec LinearFsas(Ragged<int32_t> &symbols);
 
+/* Compute the shortest distance in the tropical semiring, but
+   with the opposite sign. That is, it uses `max` instead of `min.
+
+   @param [in] fsa           The input FSA. It is on CPU and must be
```

Comment:
OK, but in general supporting FsaVec is as important, or more important, than supporting Fsa.

Reply:
Please be very clear if it must have 2 axes, i.e. if FsaVec is not supported.  The implementation code calls the 'fsa' arg FsaVec.

Reply:
This function has been removed as it does not support FsaVec.